addons/egoventure/nodes/main_menu.gd

```diff
@@ -35,7 +35,7 @@ var _selected_slot: int
 var _is_save_mode: bool
 
 # The currently visible save slot page
-var _save_slot_page: int = 1
+var _save_slot_page: int = _get_save_slot_page_last_modified()
```

Comment:
I think, this call should be better placed at the end of the `configure` function.